app/controllers/admin/merchants_controller.rb

```diff
@@ -0,0 +1,26 @@
+class Admin::MerchantsController < ApplicationController
+  def index
+    @merchants = Merchant.all
+  end
+
+  def show
+    @merchant = Merchant.find(params[:id])
+  end
+
+  def edit
+    @merchant = Merchant.find(params[:id])
+  end
+
+  def update
+    merchant = Merchant.find(params[:id])
+    merchant.update(merch_params)
+    redirect_to admin_merchant_path(merchant.id)
```

Comment:
DO we need or want a sad path for this?

Reply:
I thought so too at first, but since the field is prepopulated, even if the user doesn't make any changes, the form still technically "updates". Not sure how to differentiate those two scenarios in testing? Because if they don't change it, we don't want the user to just get stuck on the page either.

Reply:
Yeah, I think you're right.